app/src/test/java/se/inera/intyg/minaintyg/information/service/GetEnvironmentServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package se.inera.intyg.minaintyg.information.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class GetEnvironmentServiceTest {
```

Comment:
This class it currently testing that your mocking works. As discussed rewrite the test and also adapt the implementation.